examples/test_getting_started.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import pytest
+
+from examples.getting_started import getting_started
+from haystack.schema import Answer, Document
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("provider", ["cohere", "huggingface", "openai"])
```

Comment:
Did we miss this or it's intentional?
```suggestion
@pytest.mark.parametrize("provider", ["anthropic", "cohere", "huggingface", "openai"])
```

Reply:
It's intentional since I cannot create an anthropic API key for our CI (I used a personal one for testing). We will need to activate this sometime later.